fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/Options.java

```diff
@@ -346,6 +358,16 @@ public Options build() {
         }
     }
 
+    @Nullable
+    private static Object parseStringOption(@Nonnull final Name name, String valueAsString) throws SQLException {
+        for (OptionContract contract : Objects.requireNonNull(OPTIONS).get(name)) {
+            if (contract instanceof TypeContract<?>) {
```

Comment:
I believe the recently added `DISABLED_PLANNER_RULES` option does not have a type contract. Instead it has a `CollectionContract` which is sort of like a `TypeContract`, except it has a child contract to apply to each element. Given that we're not using this option in the tests, we could probably leave without this one working (for now), or we could extend this check to also allow for `CollectionContract`s